Rakefile

```diff
@@ -11,17 +11,15 @@ end
 # -----------------------------------------------------------------------------
 APP_RAKEFILE = File.expand_path("../spec/dummy/Rakefile", __FILE__)
 load "rails/tasks/engine.rake"
-Bundler::GemHelper.install_tasks
+begin
+  Bundler::GemHelper.install_tasks
+rescue RuntimeError
```

Comment:
so, we share this Rakefile for development -- should it be desired. And it will contain the basic build task. So, if it's being loaded from teaspoon-qunit, it does the right thing, but if it's being loaded from the root (aka Teaspoon), it doesn't know which gemspec to use for the release commands since there's two (teaspoon.gemspec and teaspoon-dev.gemspec)